src/components/DateRangePicker.jsx

```diff
@@ -228,9 +228,9 @@ export default class DateRangePicker extends React.Component {
 
   doesNotMeetMinimumNights(day) {
     const { startDate, focusedInput, minimumNights } = this.props;
-    const today = moment().startOf('day');
-    const dayDiff = startDate ? day.diff(startDate, 'days') : day.diff(today, 'days');
+    if (!startDate) return false;
 
+    const dayDiff = day.diff(startDate, 'days');
```

Comment:
Will this work when `startDate` is `null`? What's the behavior then?

Also this is an interesting side-effect that I had not thought about.